### PR TITLE
Added Basic PNA externs: Meter, Counter, Hash, etc 

### PR DESCRIPTION
**Pull request by rupesh-chiluka-marvell**
Part of Issue: https://github.com/p4lang/behavioral-model/issues/1245

### CODE REVIEW

**Comment by rupesh-chiluka-marvell** [comment]
@antoninbas @qobilidop. Please review the PR

File: targets/pna_nic/pnic_CLI.py [hunk 1]
```diff
@@ -48,7 +48,28 @@ def do_get_time_since_epoch(self, line):
         print(self.pnic_client.get_time_since_epoch_us())
 
 def load_json_pna(json):
-    pass
+    def get_json_key(key):
+        return json.get(key, [])
+
+    for j_meter in get_json_key("extern_instances"):
```
Comment (antoninbas) [inline targets/pna_nic/pnic_CLI.py:54]: I would remove the changes to this file from this PR. I am not sure why we are only adding support for meters, but if that's what we need, I would still do it in a follow-up PR.

**Review by antoninbas — COMMENTED** [review]
Is there any extern in this PR that is specific to PNA or different from its PSA equivalent? I am asking so I know where I should focus my review efforts.

File: targets/pna_nic/pnic_CLI.py (hunk 1 above)
Comment (rupesh-chiluka-marvell) [reply targets/pna_nic/pnic_CLI.py:54]: In this PR, No.

**Comment by rupesh-chiluka-marvell** [comment]
> Is there any extern in this PR that is specific to PNA or different from its PSA equivalent? I am asking so I know where I should focus my review efforts.

In this PR. No. These are general externs used in P4 programs.

**Review by antoninbas — APPROVED** [review]

**Pull request merged**